elasticsearch/testcontainers/elasticsearch/__init__.py

```diff
@@ -88,7 +88,7 @@ def __init__(self, image: str = "elasticsearch", port: int = 9200, **kwargs) ->
 
     @wait_container_is_ready(URLError)
     def _connect(self) -> None:
-        res = urllib.request.urlopen(self.get_url())
+        res = urllib.request.urlopen(self.get_url(), timeout=1)
```

Comment:
What is the default timeout if we don't add this? Infinity?

Reply:
i think it is infinity, yes

in the future when we add strategies and all the oop fun/gunk (i havent worked on tc-java enough to know which it is) for default waiting etc, then we can centralize this. but for now, want to proliferate it so it is not lost.

Reply:
It means without adding this, you could see behavior where we would wait indefinitely for the connection (maybe because there was an issue during container startup) and so the test suite would hang?